app/src/main/java/com/example/harvestfresh/MainActivity.java

```diff
@@ -64,7 +64,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
     @Override
     public boolean onOptionsItemSelected(@NonNull MenuItem item) {
-        if(item.getItemId() == R.id.mLogout){
+        if (item.getItemId() == R.id.mLogout) {
             onLogout();
             return true;
```

Comment:
Missed a string to const on 77...sorry, the code review tool wont let me comment more than like 3 lines out of your changes.